src/Transport/Config/AzureServiceBusTransportInfrastructure.cs

```diff
@@ -27,7 +27,19 @@ public override EndpointInstance BindToLocalEndpoint(EndpointInstance instance)
 
         public override string ToTransportAddress(LogicalAddress logicalAddress)
         {
-            return logicalAddress.ToString();
+            var queue = new StringBuilder(logicalAddress.QueueName);
```

Comment:
Why couldn't this be encapsulated in `LogicalAddress.ToString()`?
Or, alternatively, not to have a `LogicalAddress.ToToTransportAddress()`?